src/commands/bootstrap.js

```diff
@@ -29,7 +29,7 @@ module.exports = function bootstrap({ client, config }) {
 
   client.on('guildMemberRemove', (member) => {
     member.guild.channels.cache
-      .find((ch) => ch.name === 'introduction')
+      .find((ch) => ch.name === 'introductions')
```

Comment:
This might be more useful as a config setting. (this can be done in another issue/pr)

also, to prevent the server from exploding check that `.find` returns something before calling `.send`. If it **doesn't** find the channel specified log an error about how the "remove channel" wasn't found or something.

Reply:
Should be corrected now.